raport/invoice-db.php

```diff
@@ -0,0 +1,134 @@
+<?php
+require('fpdf17/fpdf.php');
+$con=mysqli_connect('localhost','root','');
```

Comment:
Powinnaś to przenieść do jakiegoś nowego repository i używać $config do łaczenia się i pobierania danych z bazy